### ISSUE TITLE
[achievement] We're Going to Need More Saddles

### ISSUE THREAD

**Issue by lavalash551** [Valhalla-Gaming/Tracker#5269]
http://www.wowhead.com/achievement=7860/were-going-to-need-more-saddles
You should get a mount for that achievement but it doesnt work.
http://www.wowhead.com/item=91802/jade-pandaren-kite-string
![wowscrnshot_050316_173652](https://cloud.githubusercontent.com/assets/16776049/14989088/7b2741e0-1156-11e6-8395-40199c34996d.jpg)
![wowscrnshot_050316_174340](https://cloud.githubusercontent.com/assets/16776049/14989110/918bf2c8-1156-11e6-96d7-fd2df166ead7.jpg)

**Comment by Collector-Valhalla** [comment]
Dayum, nice UI. How's it called? :P

**Comment by Trekex** [comment]
And gratz for achiev! :D

**Comment by Tregeta** [comment]
@Collector-Valhalla Looks like super villain ui

**Comment by lavalash551** [comment]
Correct, its SuperVillainUI @collector and thx @ Trekex but just another achiev^^ ive got almost everything done whats working. 500+ achievements and 5100+ achievement points atm.

**Comment by Collector-Valhalla** [comment]
Thanks guys, that UI looks rly nice.

I have 6540 achievement points on my druid and I'm pretty sure you can get much more now as I haven't played much on it for a long time.
![wowscrnshot_050416_142124](https://cloud.githubusercontent.com/assets/14946340/15014025/9f2bd95e-1203-11e6-9c39-68c30744283e.jpg)

**Comment by lavalash551** [comment]
...possible, but not on talador server with only only some lowlevels online. So no PvP and no draenor dungeon achievements :P
